client/package.json

```
@@ -4,6 +4,9 @@
   "description": "",
   "main": "index.js",
   "type": "module",
+  "engines": {
+    "node": ">=14.8.0"
```

```suggestion
"node": ">=15.12.0"
```
This project is using the latest version